Lecture1.hs

```diff
@@ -0,0 +1,96 @@
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
+module Lecture1
+(--makeSnippet
+ sumOfSquares
+, lastDigit
+, minmax
+, subString
+, strSum
+, lowerAndGreater
+, increase
+) where
+
+
+
+
+increase :: Integer -> Integer
+increase x = x + 1
+
+sumOfSquares :: Integer -> Integer -> Integer
+sumOfSquares n1 n2 = ( n1 ^ 2 ) + (n2 ^ 2)
+
+--finding last digit = finding the remainder when this number is divided by 10
+lastDigit :: Integer -> Integer
+lastDigit n = l
+ where
+     l = mod n 10
```

Comment:
That's almost correct! 👍🏻 

However, this function doesn't return the correct answer on negative numbers because of how `mod` works for negative numbers

```haskell
λ: mod (-17) 10
3
```